doxygen/dox/ExamplesAPI.dox

```diff
@@ -387,347 +387,345 @@ FORTRAN
-<a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/h5ex_t_floatatt.c">C</a>
-<a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/FORTRAN/H5T/h5ex_t_floatatt_F03.F90">FORTRAN</a>
-<a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/JAVA/H5T/H5Ex_T_FloatAttribute.java">Java</a>
+<a href="https://\SRCURL/HDF5Examples/C/H5T/h5ex_t_floatatt.c">C</a>
+<a href="https://\SRCURL/HDF5Examples/FORTRAN/H5T/h5ex_t_floatatt_F03.F90">FORTRAN</a>
+<a href="https://\SRCURL/HDF5Examples/JAVA/H5T/H5Ex_T_FloatAttribute.java">Java</a>
 <a href="https://github.com/HDFGroup/hdfview/blob/master/test/org.hdfgroup.object.example.test/datatypes/H5ObjectEx_T_FloatAttribute.java">JavaObj</a>
   MATLAB  PyHigh  PyLow
 </td>
 <td>h5ex_t_floatatt.h5</td>
-<td><a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/tfiles/16/h5ex_t_floatatt.tst">h5ex_t_floatatt.tst</a></td>
-<td><a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/tfiles/18/h5ex_t_floatatt.ddl">h5ex_t_floatatt.ddl</a></td>
+<td><a href="https://\SRCURL/HDF5Examples/C/H5T/tfiles/16/h5ex_t_floatatt.tst">h5ex_t_floatatt.tst</a></td>
+<td><a href="https://\SRCURL/HDF5Examples/C/H5T/tfiles/18/h5ex_t_floatatt.ddl">h5ex_t_floatatt.ddl</a></td>
 </tr>
 <tr>
 <td>Read / Write Floating Point  (Dataset)</td>
 <td>
-<a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/h5ex_t_float.c">C</a>
-<a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/FORTRAN/H5T/h5ex_t_float_F03.F90">FORTRAN</a>
-<a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/JAVA/H5T/H5Ex_T_Float.java">Java</a>
+<a href="https://\SRCURL/HDF5Examples/C/H5T/h5ex_t_float.c">C</a>
+<a href="https://\SRCURL/HDF5Examples/FORTRAN/H5T/h5ex_t_float_F03.F90">FORTRAN</a>
+<a href="https://\SRCURL/HDF5Examples/JAVA/H5T/H5Ex_T_Float.java">Java</a>
 <a href="https://github.com/HDFGroup/hdfview/blob/master/test/org.hdfgroup.object.example.test/datatypes/H5ObjectEx_T_Float.java">JavaObj</a>
   MATLAB  PyHigh  PyLow
 </td>
 <td>h5ex_t_float.h5</td>
-<td><a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/tfiles/16/h5ex_t_float.tst">h5ex_t_float.tst</a></td>
-<td><a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/tfiles/18/h5ex_t_float.ddl">h5ex_t_float.ddl</a></td>
+<td><a href="https://\SRCURL/HDF5Examples/C/H5T/tfiles/16/h5ex_t_float.tst">h5ex_t_float.tst</a></td>
+<td><a href="https://\SRCURL/HDF5Examples/C/H5T/tfiles/18/h5ex_t_float.ddl">h5ex_t_float.ddl</a></td>
 </tr>
 <tr>
 <td>Read / Write Integer Datatype (Attribute)</td>
 <td>
-<a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/h5ex_t_intatt.c">C</a>
-<a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/FORTRAN/H5T/h5ex_t_intatt_F03.F90">FORTRAN</a>
-<a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/JAVA/H5T/H5Ex_T_IntegerAttribute.java">Java</a>
```

Comment:
Looks like this link to the Java example was dropped here rather than converted to the `\SRCURL` approach

Reply:
added link in new commit, along with changes from /develop/ to /v1_14/ or /hdf5_1_14/ in some .md files.